File: PERL/Cap05/c05ex05.pl
# c05ex05.pl

#!/usr/bin/perl

use strict;
use warnings;

my $I = 1;
do {
  printf "Entre um valor numerico: ";
  my $N = <STDIN>;
  chomp $N;

  my $R = $N * 3;

  print "Resultado = $R\n";
  print "\n";

  $I = $I + 1;
} until ($I > 5);

print "\n";
print "Tecle <Enter> para encerrar... ";
<STDIN>;

File: GO/Cap05/c05ex05.go
// c05ex05.go

package main
import "fmt"

func main() {
  var I, N, R int

  I = 1
  for {
    fmt.Print("Entre um valor numerico: ")
    fmt.Scan(&N)

    R = N * 3

    fmt.Println("Resultado =", R)
    fmt.Println()

    I++
    
    if (I > 5) {break}    
  }

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
